data/datacatalogue/CatalogueOntologies/Countries.csv

```diff
@@ -1,250 +1,251 @@
-Malaysia,MY,134,,,
-Maldives,MV,135,,,
-Mali,ML,136,,,
-Malta,MT,137,,,
-Marshall Islands (the),MH,138,,,
-Martinique,MQ,139,,,
-Mauritania,MR,140,,,
-Mauritius,MU,141,,,
-Mayotte,YT,142,,,
-Mexico,MX,143,,,
-Micronesia (Federated States of),FM,144,,,
-Moldova (the Republic of),MD,145,,,
-Monaco,MC,146,,,
-Mongolia,MN,147,,,
-Montenegro,ME,148,,,
-Montserrat,MS,149,,,
-Morocco,MA,150,,,
-Mozambique,MZ,151,,,
-Myanmar,MM,152,,,
-Namibia,NA,153,,,
-Nauru,NR,154,,,
-Nepal,NP,155,,,
-Netherlands (the),NL,156,,,
-New Caledonia,NC,157,,,
-New Zealand,NZ,158,,,
-Nicaragua,NI,159,,,
-Niger (the),NE,160,,,
-Nigeria,NG,161,,,
-Niue,NU,162,,,
-Norfolk Island,NF,163,,,
-North Macedonia,MK,164,,,
-Northern Mariana Islands (the),MP,165,,,
-Norway,NO,166,,,
-Oman,OM,167,,,
-Pakistan,PK,168,,,
-Palau,PW,169,,,
-"Palestine, State of",PS,170,,,
-Panama,PA,171,,,
-Papua New Guinea,PG,172,,,
-Paraguay,PY,173,,,
-Peru,PE,174,,,
-Philippines (the),PH,175,,,
-Pitcairn,PN,176,,,
-Poland,PL,177,,,
-Portugal,PT,178,,,
-Puerto Rico,PR,179,,,
-Qatar,QA,180,,,
-Romania,RO,181,,,
-Russian Federation (the),RU,182,,,
-Rwanda,RW,183,,,
-Réunio,RE,184,,,
-Saint Barthélem,BL,185,,,
-"Saint Helena, Ascension and Tristan da Cunha",SH,186,,,
-Saint Kitts and Nevis,KN,187,,,
-Saint Lucia,LC,188,,,
-Saint Martin (French part),MF,189,,,
-Saint Pierre and Miquelon,PM,190,,,
-Saint Vincent and the Grenadines,VC,191,,,
-Samoa,WS,192,,,
-San Marino,SM,193,,,
-Sao Tome and Principe,ST,194,,,
-Saudi Arabia,SA,195,,,
-Senegal,SN,196,,,
-Serbia,RS,197,,,
-Seychelles,SC,198,,,
-Sierra Leone,SL,199,,,
-Singapore,SG,200,,,
-Sint Maarten (Dutch part),SX,201,,,
-Slovakia,SK,202,,,
-Slovenia,SI,203,,,
-Solomon Islands,SB,204,,,
-Somalia,SO,205,,,
-South Africa,ZA,206,,,
-South Georgia and the South Sandwich Islands,GS,207,,,
-South Sudan,SS,208,,,
-Spain,ES,209,,,
-Sri Lanka,LK,210,,,
-Sudan (the),SD,211,,,
-Suriname,SR,212,,,
-Svalbard and Jan Mayen,SJ,213,,,
-Sweden,SE,214,,,
-Switzerland,CH,215,,,
-Syrian Arab Republic (the),SY,216,,,
-Taiwan (Province of China),TW,217,,,
-Tajikistan,TJ,218,,,
-"Tanzania, the United Republic of",TZ,219,,,
-Thailand,TH,220,,,
-Timor-Leste,TL,221,,,
-Togo,TG,222,,,
-Tokelau,TK,223,,,
-Tonga,TO,224,,,
-Trinidad and Tobago,TT,225,,,
-Tunisia,TN,226,,,
-Turkey,TR,227,,,
-Turkmenistan,TM,228,,,
-Turks and Caicos Islands (the),TC,229,,,
-Tuvalu,TV,230,,,
-Uganda,UG,231,,,
-Ukraine,UA,232,,,
-United Arab Emirates (the),AE,233,,,
-United Kingdom of Great Britain and Northern Ireland (the),GB,234,,,
-United States Minor Outlying Islands (the),UM,235,,,
-United States of America (the),US,236,,,
-Uruguay,UY,237,,,
-Uzbekistan,UZ,238,,,
-Vanuatu,VU,239,,,
-Venezuela (Bolivarian Republic of),VE,240,,,
-Viet Nam,VN,241,,,
-Virgin Islands (British),VG,242,,,
-Virgin Islands (U.S.),VI,243,,,
-Wallis and Futuna,WF,244,,,
-Western Sahara*,EH,245,,,
-Yemen,YE,246,,,
-Zambia,ZM,247,,,
-Zimbabwe,ZW,248,,,
+order,name,code,definition,parent,ontologyTermURI
+1,Worldwide,,,,
```

Comment:
Shouldn't this option be in _Regions_?

Reply:
Regions are now more considered to be smaller portions of countries. What UMCG wants to say here is: subjects/data may come from all countries in the world.